framework/doc/content/source/transfers/MultiAppGeneralFieldTransfer.md

```diff
@@ -42,7 +42,8 @@ All transfers derived from this base class should be able to support:
 - interpolation and extrapolation transfers, as defined by the derived class
 - detection of indetermination due to source points equidistant to a target point
 - limitation of transfer source to the nearest position (see [Positions](syntax/Positions/index.md)) of target point
-
+- general coordinate transformations. Coordinate system changes (RZ to XYZ, for example) are not
+  fully supported for the the "floating point precision indetermination" detection.
```

Comment:
```suggestion
  fully supported for the "floating point precision indetermination" detection.
```
Oops completely missed this extra 'the'. Otherwise looks good.